Modules/_sqlite/connection.c

```diff
@@ -1998,10 +2003,28 @@ pysqlite_connection_iterdump_impl(pysqlite_Connection *self)
         }
         return NULL;
     }
+    PyObject *args[2] = {(PyObject *)self, filter};
+    PyObject *kwnames = PyTuple_New(1);
+    PyObject *py_filter = NULL;
 
-    PyObject *retval = PyObject_CallOneArg(iterdump, (PyObject *)self);
+    if (!kwnames) {
+        goto error;
+    }
+    py_filter = PyUnicode_FromString("filter");
+    if (!py_filter) {
+        goto error;
+    }
+    PyTuple_SET_ITEM(kwnames, 0, py_filter);
```

Comment:
Actually, let's use `Py_BuildValue` here; it will get the work done in very few lines :)

Reply:
Great shortcut, thanks :confetti_ball: